tests/StorageApiTestCase.php

```diff
@@ -562,7 +562,7 @@ protected function getExportFilePathForTest($fileName)
     /**
      * @param string $branchPrefix
      */
-    protected function deleteBranchesByPrefix(DevBranches $devBranches, $branchPrefix)
+    public function deleteBranchesByPrefix(DevBranches $devBranches, $branchPrefix)
```

Comment:
Tohle jak jsme se bavili - mnohem radši bych to viděl v nějakém izolovaném "branch provideru" ale co už :)